src/swarm-city.html

```diff
@@ -306,6 +306,9 @@
       }
 
       _loadAnimations() {
+        setTimeout(()=>{
```

Comment:
not a big fan of timeouts can we do this any other way?

Reply:
Same as the other remarks, not sure how this got in. I'll ask Xardas